### DISCUSSION TITLE
Video:" PyTorch fundementals" - video #37

### DISCUSSION THREAD

**Discussion by Anas-Odeh** [mrdbourke/pytorch-deep-learning#446] [Q&A] [answered]

**Comment by mrdbourke** [comment, marked as answer]
Hello @Anas-Odeh,

Did you have anything that needs help?

I can't see any text/pictures in your question.

**Comment by Anas-Odeh** [comment]
Hi Daniel,

I've solved the problem 🙂
Thx



---------------------------------
An energetic, young PhD candidate, Anas Odeh[💪🏻]

Department of Genetics & Developmental Biology[🧬]

Peleg Hasson Lab - Tissue Development & Regeneration[🔬]

The Ruth and Bruce Rappaport Faculty of Medicine

The Technion – Israel Institute of Technology

Cell: +972-54-9909-194








________________________________
